packages/drafts-realm/app-card.gts

```diff
@@ -0,0 +1,568 @@
+        letter-spacing: var(--boxel-lsp);
+      }
+      .grid-title {
+        margin: 0;
+        font: 500 var(--boxel-font-sm);
+        text-align: center;
+      }
+      .grid-display-name {
+        margin: 0;
+        font: 500 var(--boxel-font-xs);
+        text-align: center;
+        color: var(--grid-card-label-color);
+      }
+
+      .cards-grid {
+        --grid-card-text-thumbnail-height: 6.25rem;
+        --grid-card-label-color: var(--boxel-450);
+        --grid-card-width: 10.125rem;
+        --grid-card-height: 15.125rem;
+
+        max-width: 70rem;
+        margin: 0 auto;
+        padding: var(--boxel-sp-xl);
+        position: relative; /* Do not change this */
+        height: 100%;
+      }
+      .cards {
+        list-style-type: none;
+        margin: 0;
+        padding: 0;
+        display: grid;
+        grid-template-columns: repeat(
+          auto-fit,
+          minmax(var(--grid-card-width), 1fr)
+        );
+        gap: var(--boxel-sp);
+        justify-items: center;
+        height: 100%;
+      }
+      .cards > li {
+        height: max-content;
+      }
+      .operator-mode .buried .cards,
+      .operator-mode .buried .add-button {
+        display: none;
+      }
+
+      .add-button {
+        display: inline-block;
+        position: sticky;
+        left: 100%;
+        bottom: 10px;
+        z-index: 1;
+        margin: 10px;
+      }
+      .grid-card {
+        width: var(--grid-card-width);
+        height: var(--grid-card-height);
+        padding: var(--boxel-sp-lg) var(--boxel-sp-xs);
+      }
+      .grid-card > *,
+      .grid-thumbnail-text {
+        display: -webkit-box;
+        -webkit-line-clamp: 2;
+        -webkit-box-orient: vertical;
+        overflow: hidden;
+      }
+      .grid-thumbnail {
+        display: flex;
+        align-items: center;
+        height: var(--grid-card-text-thumbnail-height);
+        background-color: var(--boxel-teal);
+        background-position: center;
+        background-size: cover;
+        background-repeat: no-repeat;
+        color: var(--boxel-light);
+        padding: var(--boxel-sp-lg) var(--boxel-sp-xs);
+        border-radius: var(--boxel-border-radius);
+        font: 700 var(--boxel-font);
+        letter-spacing: var(--boxel-lsp);
+      }
+      .grid-title {
+        margin: 0;
+        font: 500 var(--boxel-font-sm);
+        text-align: center;
+      }
+      .grid-display-name {
+        margin: 0;
+        font: 500 var(--boxel-font-xs);
+        text-align: center;
+        color: var(--grid-card-label-color);
+      }
+      .grid-thumbnail + * {
+        margin-top: var(--boxel-sp-lg);
+      }
+      .grid-title + .grid-display-name {
+        margin-top: 0.2em;
+      }
+    </style>
+  </template>
+}
+
+export class AppCard extends CardDef {
+  static displayName = 'AppCard';
+
+  static prefersWideFormat = true;
+  static headerColor = '#009879';
+
+  @field tabs = containsMany(Tab);
```

Comment:
Looks super great. I wonder whats the future api for narrowing the fields / columns displayed inside of a table.

Reply:
Yeah I'm not sure, I think with this we could probably add things like just a string list of columns. Perhaps the tab field should contain the template for rendering the tab itself? 

It's nice that we can get this kind of functionality within a card, because then we can always have one that can deal with some basic UI, and another that provides another kind of UI without having to have "one true app card".